src/System.Text.Encodings.Web/src/System/Text/Unicode/UnicodeHelpers.cs

```diff
@@ -27,7 +27,7 @@ internal static unsafe partial class UnicodeHelpers
 
         // This field is only used on big-endian architectures. We don't
         // bother computing it on little-endian architectures.
-        private static readonly uint[] _definedCharacterBitmapBigEndian = (BitConverter.IsLittleEndian) ? null : CreateDefinedCharacterBitmapMachineEndian();
+        private static readonly uint[]? _definedCharacterBitmapBigEndian = (BitConverter.IsLittleEndian) ? null : CreateDefinedCharacterBitmapMachineEndian();
```

Comment:
Looks like the build is failing because UnicodeHelpers.cs is included in the tests project... you'll need to add `#nullable enable` at the top of this file.